app/assets/javascripts/quick_entry.js

```diff
@@ -0,0 +1,28 @@
+if($("#quick-entry").length){
+  var browserSupportFlag =  new Boolean();
+  var initialLocation;
+  if(navigator.geolocation) {
+    browserSupportFlag = true;
+    navigator.geolocation.getCurrentPosition(function(position) {
+      initialLocation = new google.maps.LatLng(position.coords.latitude,position.coords.longitude);
+      $("#entry_latitude").val(initialLocation.lat());
+      $("#entry_longitude").val(initialLocation.lng());
+      debugger;
+      $(".button").show();
+      $(".loading").hide();
+    }, function() {
+      handleNoGeolocation(browserSupportFlag);
+    });
+  }
+  else {
+    browserSupportFlag = false;
+    handleNoGeolocation(browserSupportFlag);
+  }
+  function handleNoGeolocation(errorFlag) {
```

Comment:
Function declarations should not be placed in blocks. Use a function expression or move the statement to the top of the outer function.